scenario/GovernanceScenario.ts

```diff
@@ -3,7 +3,7 @@ import { expect } from 'chai';
 import { constants, utils } from 'ethers';
 import { CometModifiedFactory, CometModifiedFactory__factory } from '../build/types';
 
-scenario('upgrade Comet implementation and initialize', { upgradeAll: true }, async ({ comet, configurator, proxyAdmin }, world, context) => {
+scenario.skip('upgrade Comet implementation and initialize', { upgradeAll: true }, async ({ comet, configurator, proxyAdmin }, world, context) => {
```

Comment:
I'm skipping these two specifically for now because I want these to test the governance flow. Planning on supporting execution of mainnet governance proposals in a separate PR.